### DISCUSSION TITLE
Chameleon Light and Dark Themes

### DISCUSSION THREAD

**Discussion by DavidFuchs** [TriliumNext/Trilium#3915] [Show and tell]
I've made a matching light and dark theme for Trilium called 'Chameleon'. The name stems from the fact that the accent colour, as well as accent/text/background brightness is easy to adjust with a couple variables the user can tweak.

This theme doesn't alter much HTML, except where I felt it was necessary (generally because something looked off and there was no variable to control it).

Please give it a try and let me know your thoughts!

[https://github.com/DavidFuchs/trilium-chameleon-theme](https://github.com/DavidFuchs/trilium-chameleon-theme)

**Comment by zadam** [comment]
Very nice!

**Comment by Nriver** [comment]
Cool
